### PR TITLE
WIP: Button

### PR DESCRIPTION
**Pull request by balta-z-r**

### CODE REVIEW

**Comment by alanwu4321** [comment]
da ripple effect though 😫❤️

File: src/Button/styles.ts [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    }
+  }};
+  color: ${({ ghost, disabled, theme }) => {
+    if (disabled) return theme.colors.gray5
+    if (ghost) {
+      return theme.colors.primary
+    } else {
+      return 'white'
+    }
+  }};
```
Comment (ian13456) [inline src/Button/styles.ts:76]: Add an empty line below to separate out different type of styles.

File: src/Button/styles.ts [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+  color: ${({ ghost, disabled, theme }) => {
+    if (disabled) return theme.colors.gray5
+    if (ghost) {
+      return 'white'
+    } else {
+      return theme.colors.gray7
+    }
+  }};
```
Comment (ian13456) [inline src/Button/styles.ts:166]: Same here.

File: src/Button/styles.ts [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+    return 'white'
+  }};
+  color: ${({ ghost, disabled, theme }) => {
+    if (disabled) return theme.colors.gray5
+    if (ghost) {
+      return 'white'
+    } else {
+      return theme.colors.gray7
+    }
+  }};
```
Comment (ian13456) [inline src/Button/styles.ts:197]: Same here.

File: src/Button/styles.ts [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+    return 'white'
+  }};
+  color: ${({ ghost, disabled, theme }) => {
+    if (disabled) return theme.colors.gray5
+    if (ghost) {
+      return 'white'
+    } else {
+      return theme.colors.primary
+    }
+  }};
```
Comment (ian13456) [inline src/Button/styles.ts:238]: You get the idea.

File: src/Button/styles.ts [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+export const RippleSpan = styled.span<RippleProps>`
+  position: absolute;
+  background: #555;
+  transform: translate(-50%, -50%);
+  left: ${({ x }) => rem(`${x}px`)};
+  top: ${({ y }) => rem(`${y}px`)};
+  pointer-events: none;
+  border-radius: 50%;
+  animation: ${({ width, top, right, bottom, left, shape }) => {
+      return keyframes`
```
Comment (ian13456) [inline src/Button/styles.ts:268]: Would be nice to extract the keyframes?

**Pull request merged**